src/HydrationRepository.js

```diff
@@ -16,15 +20,15 @@ class HydrationRepository {
     }, 0) / this.user.length);
   }
 
-  totalOzDay(date) {
-    let foundUser = this.user.find(user => user.date === date);
-    return foundUser.numOunces;
+  totalOzDay(id, date, dataType) {
+    console.log(dataType);
+    return this.getUserInfoByDate(id, date, dataType).numOunces;
```

Comment:
A great example of inheritance - we no longer need any logic for finding the user, since we are handling that with the inherited method in the constructor on line 8. We just call the inherited method, tack on the numOunces, and we are good to go! Slick!